artificial-scrum-master/apps/artificial.scrum.master.frontend/src/app/features/estimation-poker/services/estimation-poker.service.ts

```diff
@@ -42,16 +45,32 @@ export class EstimationPokerService {
   private readonly newTaskLoadedSubject$ = new Subject<void>();
   public newTaskLoaded$ = this.newTaskLoadedSubject$.asObservable();
 
+  #userStories = signal<UserStoryPreview[]>([]);
+  public userStories = this.#userStories.asReadonly();
+
+  public isLoading = signal<boolean>(false);
```

Comment:
do czego służy to isLoading i czemu ustawiamy je poza tą klasą?

Reply:
Miał to być taki globalny loader ale trochę nie wyszło, przemyśle jak to ogarnąć lepiej xD

Reply:
Wyciągne to do serwisu popupa